### PR TITLE
Location page metadata

### PR DESCRIPTION
**Pull request by sethvincent**
Changes include:

- use `metadata=true` query param to fetch metadata to show on location view
- adds metadata section on location view
- add link to metadata.openaq.org so people can see more metadata and make edits there
- uses openaq-data-format to determine metadata fields to show
- a handful of small edits to guard against missing data (this happened in dev environment, should not impact staging or production in any way)

### CODE REVIEW

**Comment by sethvincent** [comment]
The bundler error happening in ci looks like a similar issue where we needed to move to uglify-es instead of using uglify-js. I'll add a commit to fix that.

File: app/assets/scripts/views/location.js [hunk 1]
```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
...
+      }
+      if (prop.type && prop.type === 'boolean') {
+        val = val ? 'Yes' : 'No';
+      }
+
+      const sectionIndex = Math.floor(i / length);
+
+      switch (sectionIndex) {
+        case 0: {
+          propertiesMain.push(<dt key={`${key}-${prop.title}`}>{`${prop.title}`}</dt>);
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: Unnecessary template literal around `prop.title`
```suggestion
          propertiesMain.push(<dt key={`${key}-${prop.title}`}>{prop.title}</dt>);
```

File: app/assets/scripts/views/location.js [hunk 2]
```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
...
+          propertiesMain.push(<dd key={`${key}-${prop.title}-val`}>{val}</dd>);
+          break;
+        }
+        case 1: {
+          propertiesSec.push(<dt key={`${key}-${prop.title}`} style={{ marginBottom: 0 }}>{`${prop.title}`}</dt>);
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: The `style` should probably be moved to styles.
also:
```suggestion
          propertiesSec.push(<dt key={`${key}-${prop.title}`} style={{ marginBottom: 0 }}>{prop.title}</dt>);
```

File: app/assets/scripts/views/location.js [hunk 3]
```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
...
+          propertiesSec.push(<dd key={`${key}-${prop.title}-val`}>{val}</dd>);
+          break;
+        }
+      }
+    });
+
+    return (
+      <section className='fold' id='location-stats'>
+        <div className='inner'>
+          <header className=''>
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: Empty property
```suggestion
          <header>
```

File: app/assets/scripts/views/location.js [hunk 4]
```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
...
+          <div className='col-sec'>
+            <dl>
+              {propertiesSec}
+            </dl>
+          </div>
+
+        </div>
+        <div className='inner' style={{ marginTop: 30 }}>
+          <p>
+            Have more information about this location? <a href={`${config.metadata}/location/${loc.data.id}`}>Update the metadata</a>
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: The link should have a `title` attribute

File: app/assets/scripts/views/location.js [hunk 5]
```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
...
+            </dl>
+          </div>
+          <div className='col-sec'>
+            <dl>
+              {propertiesSec}
+            </dl>
+          </div>
+
+        </div>
+        <div className='inner' style={{ marginTop: 30 }}>
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: The `style` should probably be moved to styles.

File: app/assets/scripts/views/location.js [hunk 6]
```diff
@@ -274,22 +354,29 @@ var Location = React.createClass({
...
             </div>
-            <div className='col-sec'>
-              {sources[0].description ? <p>{sources[0].description}</p> : null}
-              For more information contact <a href={`mailto:${sources[0].contacts[0]}`}>{sources[0].contacts[0]}</a>.
-            </div>
+            {
+              sources[0] && (
+                <div className='col-sec'>
+                  {sources[0].description ? <p>{sources[0].description}</p> : null}
+                  For more information contact <a href={`mailto:${sources[0].contacts[0]}`}>{sources[0].contacts[0]}</a>.
```
Comment (danielfdsilva) [inline app/assets/scripts/views/location.js]: The link should have a `title` attribute

**Comment by sethvincent** [comment]
@danielfdsilva thanks! I made those fixes.

File: package.json [hunk 7]
```diff
@@ -85,10 +84,12 @@
     "chroma-js": "^1.2.1",
     "classnames": "^2.2.5",
     "d3": "^4.2.1",
+    "gulp-uglify-es": "^1.0.4",
     "isomorphic-fetch": "^2.2.1",
     "lodash": "^4.17.11",
     "mapbox-gl": "^0.50.0",
     "moment": "^2.14.1",
+    "openaq-data-format": "github:openaq/openaq-data-format#e3bda716006e1f69c82b8b1e98e5614688f4718e",
```
Comment (jflasher) [inline package.json]: This may just be like this for testing, but should we pin this to a commit or to something like `master`?

Comment (sethvincent) [reply package.json]: Updated this to have it track master now that the pr on openaq-data-format is merged in.

**Pull request merged**